server/server.js

```diff
@@ -167,6 +168,28 @@ app.get(`${apiBaseUrl}/participant`,
     return res.json(result);
   }));
 
+app.get(`${apiBaseUrl}/site-participants`,
```

Comment:
I would probably actually do this under the existing endpoint, e.g.

/employer-sites/:id/participants

Reply:
@Zenen-freshworks is this endpoint still supposed to exist? Noticed most of the work duplicated at :494.